kuzzleio/kuzzle-admin-console
# Add map view

## What does this PR do ?
resolve #762 

### How should this be manually tested?
Use the following `script.sh` to generate fixtures
Then check the view map on the different collections
Also try to change the geopoint field mapping on view map 
Then select a marker to see document content
```
#!/bin/bash
kourou index:delete testindex
kourou index:create testindex
kourou collection:create testindex mapcollectiontest '{"mappings": {"properties": {"location": {"type": "geo_point"}, "nestedLocation": {"properties": {"locationBis": {"type": "geo_point"}}}}}}'
kourou collection:create testindex testempty '{"mappings": {"properties": {"location": {"type": "geo_point"}}}}'
kourou collection:create testindex testnogeopoint '{"mappings": {"properties": {"age": {"type": "integer"}}}}'
documents="{documents:["
for i in {1..50}
do
  lat=$(shuf -i 43-48 -n 1)
  lon=$(shuf -i 0-6 -n 1)
  documents="$documents {body: {location: {lat: $lat.$i, lon: $lon.$i}, nestedLocation: {locationBis: {lat: $lon.$i, lon: $lat.$i}}}},"
done
documents="${documents::-1}]}"

kourou document:mCreate testindex mapcollectiontest "$documents"
```

### Other changes
/

### Boyscout
/

### Screenshots (if appropriate)
/
